src/main/java/org/minions/devfund/cristhian_arispe/CountCharacters.java

```diff
@@ -0,0 +1,32 @@
+package org.minions.devfund.cristhian_arispe;
+
+/**
+ * This is the solution for the first kata.
+ */
+public final class CountCharacters {
+
+    /**
+     * CountCharacters constructor.
+     */
+    private CountCharacters() {
+    }
+
+    /**
+     * Count characters on the String.
+     * @param str String value.
+     * @param c Char value.
+     * @return The number of characters that are found in the string.
+     */
+    public static int charCount(final String str, final char c) {
+        String[] wordArray = str.split("");
+        int counter = 0;
+        for (int i = 0; i < str.length(); i++) {
+            if (wordArray[i].toLowerCase().equals((c + "").toLowerCase())) {
+                counter++;
+            }
+        }
+        return counter;
```

Comment:
This may be refactored like this.
```java
    static int charCount(final String str, final char c) {
        int counter = 0;
        char[] wordArray = str.toLowerCase().toCharArray();
        for (char aWordArray : wordArray) {
            if (aWordArray == Character.toLowerCase(c)) {
                counter++;
            }
        }
        return counter;
    }
```